pyproject.toml

```diff
@@ -1,6 +1,6 @@
 [tool.poetry]
 name = "lksearch"
-version = "1.1.0"
+version = "1.2.0"
```

Comment:
You also need to update this in the __init__.py file to make sure the version numbers match. Alternatively you can do something like this: https://github.com/tessgi/tesswcs/blob/086509b2aa691c2aa2f6791c5846e87f79be3d84/src/tesswcs/__init__.py#L6